pkgs/development/tools/misc/cwebbin/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, fetchurl, tie }:
+
+stdenv.mkDerivation rec {
+  name = "cwebbin-${version}";
+  version = "22p";
+
+  src = fetchFromGitHub {
+    owner = "ascherer";
+    repo = "cwebbin";
+    rev = name;
+    sha256 = "0zf93016hm9i74i2v384rwzcw16y3hg5vc2mibzkx1rzvqa50yfr";
+  };
+
+  cweb = fetchurl {
+    url = https://www.ctan.org/tex-archive/web/c_cpp/cweb/cweb.tar.gz;
+    sha256 = "1hdzxfzaibnjxjzgp6d2zay8nsarnfy9hfq55hz1bxzzl23n35aj";
+  };
+
+  buildInputs = [ tie ];
+
+  buildPhase = ''
+    sed -ie 's#/usr/local#$(out)#g' Makefile*
+    sed -ie 's#/usr#$(out)#g' Makefile*
+    zcat ${cweb} | tar -xvpf -
+    make -f Makefile.unix boot
+    make -f Makefile.unix cautiously CP=cp RM=rm DESTDIR=$out/bin MANDIR=$out/share/man/man1 DESTPREF=c
```

Comment:
What about factoring out make arguments used both here and under installation into `makeFlags`?